safe/impact_functions/earthquake/itb_earthquake_fatality_model/impact_function.py

```diff
@@ -144,13 +144,10 @@ def run(self):
 
         displacement_rate = self.hardcoded_parameters['displacement_rate']
 
-        # Extract input layers
-        intensity = self.hazard
-        population = self.exposure
-
         # Extract data grids
-        hazard = intensity.get_data()   # Ground Shaking
-        exposure = population.get_data(scaling=True)  # Population Density
+        hazard = self.hazard.layer.get_data()   # Ground Shaking
```

Comment:
Some part of the java world :) . hahaha...
This is just for discussion, but what about ```get_``` prefixes in global method, not as an object getter/setter? Is it also pythonic to leave the ```get_``` out?